### DISCUSSION TITLE
Theia IDE 1.53 preview testing

### DISCUSSION THREAD

**Discussion by jfaltermeier** [eclipse-theia/theia#14123] [Release Announcements] [answered]
The new version 1.53.100 of the Theia IDE is available on the preview channel now, please join the preview testing! You can download it here: [Linux](https://download.eclipse.org/theia/ide-preview/1.53.100/linux/TheiaIDE.AppImage), [Mac](https://download.eclipse.org/theia/ide-preview/1.53.100/macos/TheiaIDE.dmg), [Windows](https://download.eclipse.org/theia/ide-preview/1.53.100/windows/TheiaIDESetup.exe). You can update your existing installation by setting the preference updates.channel to preview. Please respond here when you were able to test the preview without finding blockers, by commenting with a ✔️. If you find any issues, please mention them in this thread and report them as an issue once confirmed by other testers.

**Comment by tsmaeder** [comment]
It seems like https://github.com/eclipse-theia/theia/issues/14072 is now happening for toolbar actions like "commit" in 1.53.1, as well. I'll investigate.

**Comment by tsmaeder** [comment]
I can probably do a workaround for the problem noonish tomorrow.

**Comment by msujew** [comment]
I've tested the new collaboration feature and everything seems to work as expected :)

**Comment by tsmaeder** [comment]
I've proposed a fix for #14072 [here](https://github.com/eclipse-theia/theia/pull/14132)

**Comment by rschnekenbu** [comment]
✔️
I could not find any blocker while testing (except 14072 of course).

**Comment by jfaltermeier** [comment]
A new version [1.53.200](https://download.eclipse.org/theia/ide-preview/1.53.200) with a fix for 14072 is available now for testing

**Comment by tsmaeder** [comment]
I've been using the new version and I haven't found any new breakages. 1.53.200 is a go, as far as I'm concerned. Let's promote it on Monday morning, unless someone has an objection.

**Reply by jfaltermeier** [reply]
I haven't found any new issues as well ✔️

**Reply by sgraband** [reply]
Tested as well and no new issues found (+ the fix above is working) :heavy_check_mark:

**Reply by sgraband** [reply, marked as answer]
1.53.200 is promoted to the stable channel

**Comment by zhouxinghong** [comment]


**Reply by sgraband** [reply]
@zhouxinghong The path that is trying to be downloaded is wrong (https://download.eclipse.org/theia/ide/1.53.200/windows/latest.yml) It should be https://download.eclipse.org/theia/ide-preview/1.53.200/windows/latest.yml). I am unsure where the wrong path comes from.. What was you previous Theia IDE version that you tried to update from? Did you somehow manually change the code?

You can enable getting the previews by setting the `updates.channel` preference to `preview`.

**Reply by sgraband** [reply]
I do not understand, where you get the error when you installed 1.53.200 directly? Could you elaborate a bit more?

**Comment by JonasHelming** [comment]
[News & Noteworthy](https://eclipsesource.com/blogs/2024/09/09/eclipse-theia-1-53-release-news-and-noteworthy/)
